irmaclient/session.go

```diff
@@ -326,7 +326,7 @@ func (session *session) processSessionInfo() {
 		issuedAt := time.Now()
 		_, err := ir.GetCredentialInfoList(session.client.Configuration, session.Version, issuedAt)
 		if err != nil {
-			session.fail(&irma.SessionError{ErrorType: irma.ErrorUnknownIdentifier, Err: err})
+			session.fail(err.(*irma.SessionError))
```

Comment:
If `err` is not nil then it will not necessarily be of type `*irma.SessionError` (e.g. `requests.go` line 643), so this type assertion can cause a panic. It would be better to check if the error is this type; if so just pass it on; if not wrap it into one.